src/app/Shared/Redux/DashboardConfigReducer.tsx

```diff
@@ -40,15 +40,20 @@ import { getFromLocalStorage } from '@app/utils/LocalStorage';
 import { createReducer } from '@reduxjs/toolkit';
 import { addCardIntent, deleteCardIntent } from './DashboardConfigActions';
 
+export interface CardConfig {
+  name: string;
+  props: any;
+}
+
 // Initial states are loaded from local storage if there are any
 const initialState = {
-  list: getFromLocalStorage('DASHBOARD_CFG', []) as string[],
+  list: getFromLocalStorage('DASHBOARD_CFG', []) as CardConfig[],
 };
 
 export const dashboardConfigReducer = createReducer(initialState, (builder) => {
   builder
     .addCase(addCardIntent, (state, { payload }) => {
-      state.list.splice(payload.idx || 0, 0, payload.name);
+      state.list.splice(0, 0, payload);
```

Comment:
Ahh one thing i was meant to ask: Seems like we are adding it to the front. Should we add to the end instead?

It's just closer to the `AddCard` so the user can see them being "appended". If we have a long stack of cards, the user might wonder where the added card goes since current scrolling position is kept.

Reply:
Sure, makes sense in this stack layout and easy enough to change.